API/RestSharpAPI/Steps/BookingsApiSteps.cs

```diff
@@ -0,0 +1,283 @@
+using RestSharpApi.Data;
+using RestSharpApi.Hooks;
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using TechTalk.SpecFlow;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using NUnit.Framework;
+using Bookings;
+using Notification;
+using User;
+using Video;
+using System.Collections.Generic;
+
+namespace RestSharpApi.Steps
+{
+    [Binding]
+    public class BookingsApiSteps : RestSharpHooks
+
+    {
+        private static BookingsApi BookingApiService;
+        protected static ScenarioContext _scenarioContext;
+        private static string HearingScenarionKey = "Hearing";
+
+        [When(@"I check BookingsApi health")]
+        public void WhenICheckBookingsApiHealth()
+        {
+            throw new PendingStepException();
+        }
+
+
+        public BookingsApiSteps(ScenarioContext scenarioContext)
+        {
+            var _client = new HttpClient();
+            _client.DefaultRequestHeaders.Accept
+                .Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            _client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("Bearer", GetServiceToServiceToken());
+             BookingApiService = new BookingsApi(_client);
+            BookingApiService.BaseUrl = "https://vh-bookings-api-dev.azurewebsites.net/";
```

Comment:
I think BaseUrl should be read from appsettings.json

Reply:
fixed